File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/readmodel/QueryServices/Payment.cs
using System;

namespace Registration.ReadModel
{
    public class Payment
    {
        public Guid Id { get; set; }
        public Guid OrderId { get; set; }
        public Guid ConferenceId { get; set; }
        public int State { get; set; }
        public int TotalAmount { get; set; }
        public string Description { get; set; }
    }
}

File: samples/conference/registration-bc/src/main/java/com/microsoft/conference/registration/readmodel/service/Payment.java
package com.microsoft.conference.registration.readmodel.service;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Payment {
    private String id;
    private String orderId;
    private String conferenceId;
    private int state;
    private int totalAmount;
    private String description;
}
